core/trino-main/src/main/java/io/trino/Session.java

```diff
@@ -395,20 +382,17 @@ public Session beginTransactionId(TransactionId transactionId, TransactionManage
                 Optional.of(isAutoCommitContext));
     }
 
-    public Session withDefaultProperties(Map<String, String> systemPropertyDefaults, Map<String, Map<String, String>> catalogPropertyDefaults)
+    public Session withDefaultProperties(AccessControl accessControl, Map<String, String> systemPropertyDefaults, Map<String, Map<String, String>> catalogPropertyDefaults)
```

Comment:
minor: Why do you place `accessControl` parameter on the 1st position?
The method's name is `withDefaultProperties`, so I'd place `accessControl` actually (from precedence) on the last parameter position

Reply:
Applied